db/migrate/20160430092324_add_default_false_value_to_notifications.rb

```diff
@@ -1,5 +1,8 @@
 class AddDefaultFalseValueToNotifications < ActiveRecord::Migration
```

Comment:
@ArtOfCode- Y are right. no problem when I run `rake db:schema:load` so I restored the migrations as you had them https://github.com/fabriziobertoglio1987/railsmentor/commit/95df42dada12853633e5dece65b6787f3d725344